app/models/message.rb

```diff
@@ -61,6 +61,10 @@ def push_event_data
     data
   end
 
+  def reportable?
+    !activity?
```

Comment:
There is a new message_type `template` please account that as well here.

Reply:
I am not able to see any type called `template` in message type enum. Instead of blacklisting logic I changed `reportable` logic to whitelist. Hope this works

Reply:
https://github.com/chatwoot/chatwoot/blob/develop/app/models/message.rb#L32

Please see this line.